scripts/local_testnet/beacon_node.sh

```diff
@@ -13,6 +13,7 @@ DEBUG_LEVEL=${4:-info}
 exec lighthouse \
 	--debug-level $DEBUG_LEVEL \
 	bn \
+	--subscribe-all-subnets \
```

Comment:
Not sure why this option was added. I don't think this should be present by default.